pipelines/maven-zip-build/patch.yaml

```diff
@@ -0,0 +1,125 @@
+---
+- op: add
+  path: /spec/description
+  value: |
+    This pipeline will build the maven zip to oci-artifact.
+
+    _Uses `prefetch-dependencies` to fetch all artifacts which will be the content of the maven zip, and then uses `build-maven-zip` to create zip and push it to quay.io as oci-artifact.
+    This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/repository/konflux-ci/tekton-catalog/pipeline-maven-zip-build?tab=tags)_
+- op: replace
+  path: /metadata/name
+  value: maven-zip-build
+- op: replace
+  path: /metadata/labels
+  value:
+    "pipelines.openshift.io/used-by": "build-cloud"
+    "pipelines.openshift.io/runtime": "generic"
+    "pipelines.openshift.io/strategy": "maven-zip"
+# Remove unused parameters from the template
+# $ yq ".spec.params.[].name"  pipelines/template-build/template-build.yaml | nl -v 0
+#      0  git-url
+#      1  revision
+#      2  output-image
+#      3  path-context
+#      4  dockerfile
+#      5  rebuild
+#      6  skip-checks
+#      7  hermetic
+#      8  prefetch-input
+#      9  image-expires-after
+#     10  build-source-image
+#     11  build-image-index
+
+- op: remove
+  path: /spec/params/11  # build-image-index
+- op: remove
+  path: /spec/params/10  # build-source-image
+- op: remove
+  path: /spec/params/7  # hermetic
+- op: remove
+  path: /spec/params/4  # dockerfile
+- op: remove
+  path: /spec/params/3  # path-context
+# Change default value of skip-checks
+- op: replace
+  path: /spec/params/4/default
+  value: true
```

Comment:
Have you tried running the pipeline? I think this is invalid

Reply:
I've tested the pipeline in my local workspace with corresponding changes and it can pass.

Reply:
That's weird :thinking: 

```
$ oc kustomize pipelines/maven-zip-build | oc create -f -
Error from server (InternalError): error when creating "STDIN": Internal error occurred: Internal error occurred: conversion webhook for tekton.dev/v1, Kind=Pipeline failed: unable to unmarshal input: json: cannot unmarshal array into Go struct field PipelineSpec.spec.tasks of type v1.PipelineTask
```

(which is probably about something other than this boolean, but it doesn't work)

Reply:
Ok let me have a check.

Reply:
Ok, I think this is cause by a typo in build-maven-zip task adding. An extra dash is added there. I've fixed it in new commit.